README.md

```diff
@@ -4,9 +4,12 @@ Join us on our mission to make it so everyone, in every community, in every coun
 
 We're hiring! Check out our open positions:
 
-- [Software Engineer](/SOFTWARE_ENGINEER_JOB_DESCRIPTION.md)
-- [Product Designer](/PRODUCT_DESIGNER_JOB_DESCRIPTION.md)
-- [Developer Advocate](/DEVELOPER_ADVOCATE_JOB_DESCRIPTION.md)
+- [Software Engineer](job-descriptions/software-engineer.md)
+  - Frontend
+  - Backend
+  - Operations
```

Comment:
You know this better than I do — but would this benefit from being more descriptive, now that it's just a bullet, rather than an actual title? E.g. "Operations, tooling, and infrastructure" or something?

Reply:
The description will come in a followup PR that modifies the contents of software-engineer.md itself